source/versions/versioning.rst

```diff
@@ -0,0 +1,118 @@
+==========
+Versioning
+==========
+
+.. _sem-ver:
+
+SemVer
+======
+
+Sponge uses the semantic versioning (https://semver.org/) specification. As a result, we increment versions according 
+to the rules of SemVer every time that we make a release.
+
+SemVer uses the scheme ``X.Y.Z``, where ``X`` is a *major* version, ``Y`` is a *minor* version and ``Z`` is a *patch* 
+version. A release containing changes which are not backwards-compatible must be one *major* version ahead of the 
+previous release. If there are only new features that are still backwards compatible, then the new release will be 
+one *minor* version ahead of the previous release, and if the release strictly contains bugfixes then only the *patch* 
+version will be incremented.
+
+This means that ``3.2.0`` is fully compatible to ``3.0.0`` while ``4.0.0`` is not compatible to ``3.0.0``, ``3.1.0``, 
+or ``3.1.2``. These versions are fully compatible also; the difference between them are new features (``3.0.0`` -> 
+``3.1.0``) and bug fixes (``3.1.0`` -> ``3.1.1`` -> ``3.1.2``). ``3.2.0`` introduced more new features beyond those 
+introduced by ``3.1.0``.
+
+Version String
+==============
+
+Applying the versioning policy to Sponge begins with SpongeAPI. It is the core on which the version string is based. 
+SpongeAPI is the actual API that allows plugins to interact with the game. However, the API is enabled by the Sponge 
+implementation (SpongeForge or SpongeVanilla). Together, these products form the foundation of a Sponge server.
+
+As such, the version string reflects the state of these two products. The major and minor release components of a 
+version is based on the API while the patch component of the version is based on the implementation. Let's consider 
+this version string as an example: ``8.2.0``. This string conveys the API is Version 8.2 and the implementation is the 
+first release using this API version.
+
+The following table provides several examples of a version string and its components:
```

Comment:
This table does not help much - it may actually add to confusion. I'm not sure dissecting the semver this way is really assisting. Maybe directing via to a use-case might improve it.

Reply:
What about linking to the website like github does ? 
https://semver.org/

Reply:
Changed